pkg/epinio/pages/c/_cluster/dashboard.vue

```diff
@@ -0,0 +1,252 @@
+<script lang="ts">
+import { getVersionInfo } from '@shell/utils/version';
+import Vue from 'vue';
+import DashboardCard from '../../../components/dashboard/Cards.vue';
+import { createEpinioRoute } from '@/pkg/epinio/utils/custom-routing';
+import { EpinioApplicationResource, EpinioCatalogService, EPINIO_TYPES } from '@/pkg/epinio/types';
+import ConsumptionGauge from '@shell/components/ConsumptionGauge.vue';
+
+export default Vue.extend<any, any, any, any>({
+  components: { DashboardCard, ConsumptionGauge },
+  fetch() {
+    this.$store.dispatch(`epinio/findAll`, { type: EPINIO_TYPES.CATALOG_SERVICE });
+  },
+  data() {
+    return {
+      sectionContent: [
+        {
+          title:       this.t('typeLabel.namespaces', { count: this.namespaces?.totalConfigurations }),
```

Comment:
Hadn't quite spotted this, and might have made a bade suggestion before. The type labels don't have input for a `count`. If we want to show them then we need to have custom entries with the count appended. There's a way to reference other strings, i can't remember the syntax but it could be something like

`[typeLabel.namespaces] ({count})`